### PR TITLE
Add DateAdded as sort method, setting for default sort, added to plot

### PR DESCRIPTION
**Pull request by codingPF**
allow sorting by dateAdded which is a timestamp (inc. hours) and enables sorting by date and hours as per request in #95 
Adding airdate (timesetamp) to the plot which is the only way to show this information in Kodi at the moment.
allow to set default sorting via settings #71

### CODE REVIEW

**Comment by alex1702** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5345569&bid=17501511)

File: resources/lib/filmui.py [hunk 1]
```diff
@@ -40,12 +41,23 @@ def __init__(self, plugin, sortmethods=None):
...
+            xbmcplugin.SORT_METHOD_DURATION
         ]
+        if sortmethods is not None:
+            self.sortmethods = sortmethods
+        else:
+            method = allSortMethods[0]
+            allSortMethods[0] = allSortMethods[self.settings.filmSortMethod]
+            allSortMethods[self.settings.filmSortMethod]=method
+            self.sortmethods = allSortMethods
+
```
Comment (alex1702) [inline resources/lib/filmui.py:60]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5345569)

File: resources/lib/filmui.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import os
 
 # pylint: disable=import-error
+import xbmc
```
Comment (alex1702) [inline resources/lib/filmui.py:12]: Codacy Issue found: [Unused import xbmc](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5345569)

File: resources/language/resource.language.de_de/strings.po [hunk 3]
```diff
@@ -62,6 +62,37 @@ msgctxt "#30117"
 msgid "Recent calculated by"
 msgstr "Neuigkeit berechnet ab"
 
+<<<<<<< Upstream, based on origin/SortByDateadded
+=======
+msgctxt "#30118"
+msgid "Up-to-Date check interval [sec]"
+msgstr "Nach Updates suchen alle [sec]"
+
+>>>>>>> 066084e Add DateAdded as sort method, setting for default sort, added to plot
```
Comment (alex1702) [inline resources/language/resource.language.de_de/strings.po:71]: Merge conflict?

File: resources/language/resource.language.de_de/strings.po [hunk 4]
```diff
@@ -62,6 +62,30 @@ msgctxt "#30117"
...
+msgid "Sort by title"
+msgstr "Titel"
+
+msgctxt "#30121"
+msgid "Sort by date"
+msgstr "Sendedatum"
+
+msgctxt "#30122"
+msgid "Sort by added"
+msgstr "Hinzugef�gt"
```
Comment (alex1702) [inline resources/language/resource.language.de_de/strings.po:79]: Kodierungsproblem?

Comment (codingPF) [reply resources/language/resource.language.de_de/strings.po]: Ich habe  meine neuen Entwicklungsumgebung hier noch nicht ganz im Griff, sorry

**Pull request merged**